wordpress-mobile/WordPress-FluxC-Android
# Adds group field in the starter designs response

`WordPress-Android` PR:https://github.com/wordpress-mobile/WordPress-Android/pull/16509

⚠️ Please merge along the Android PR

## Description
This PR adds the new `group` field (ref: `D80005`) in the starter designs response. [This intents to be used to filter and recommend designs based on the vertical selection of the user during site creation](https://github.com/wordpress-mobile/WordPress-Android/issues/16503).

## To test
Check the Site Design Revamp Android PR: https://github.com/wordpress-mobile/WordPress-Android/pull/16510
